### ISSUE TITLE
Inpaint feature ceased working since yesterday in Fooocus Colab

### ISSUE THREAD

**Issue by Goliat48** [lllyasviel/Fooocus#2283]
**Read Troubleshoot**

[x] I confirm that I have read the [Troubleshoot](https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md) guide before making this issue.

**Describe the problem**

 Yesterday the inpaint function in Fooocus stop working. I tried other models, using the propietary version from Google, as well as a copy in my Google Drive. and the problem persists with different models.
I ¨m new to AI but never had that issue before.
I create an image (man in a street) , then put that image in the inpaint window and mask a small zone using the "Modify Content (add objects, change background, etc.)" option.  After clicking in "Generate" the small symbol begans to spin but no image is created. Besides the Skip and Stop buttons no longer work  meanwhile the counter clock runs indefinitely.

**Full Console Log**

Downloading: "https://huggingface.co/lllyasviel/misc/resolve/main/vaeapp_sd15.pt" to /content/Fooocus/models/vae_approx/vaeapp_sd15.pth

100% 209k/209k [00:00<00:00, 7.51MB/s]
Downloading: "https://huggingface.co/lllyasviel/misc/resolve/main/xl-to-v1_interposer-v3.1.safetensors" to /content/Fooocus/models/vae_approx/xl-to-v1_interposer-v3.1.safetensors

100% 6.25M/6.25M [00:00<00:00, 78.7MB/s]
Downloading: "https://huggingface.co/lllyasviel/misc/resolve/main/fooocus_expansion.bin" to /content/Fooocus/models/prompt_expansion/fooocus_expansion/pytorch_model.bin

100% 335M/335M [00:01<00:00, 239MB/s]
Downloading: "https://huggingface.co/lllyasviel/fav_models/resolve/main/fav/juggernautXL_v8Rundiffusion.safetensors" to /content/Fooocus/models/checkpoints/juggernautXL_v8Rundiffusion.safetensors

100% 6.62G/6.62G [00:37<00:00, 191MB/s]
Downloading: "https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0/resolve/main/sd_xl_offset_example-lora_1.0.safetensors" to /content/Fooocus/models/loras/sd_xl_offset_example-lora_1.0.safetensors

100% 47.3M/47.3M [00:00<00:00, 156MB/s]
Total VRAM 15102 MB, total RAM 12979 MB
Set vram state to: NORMAL_VRAM
Always offload VRAM
Device: cuda:0 Tesla T4 : native
VAE dtype: torch.float32
Using pytorch cross attention
Running on local URL:  http://127.0.0.1:7865
Running on public URL: https://dd82cd31ebbd71f74a.gradio.live

This share link expires in 72 hours. For free permanent hosting and GPU upgrades, run `gradio deploy` from Terminal to deploy to Spaces (https://huggingface.co/spaces)
2024-02-17 21:08:07.476803: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:9261] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2024-02-17 21:08:07.476855: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:607] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2024-02-17 21:08:07.478419: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1515] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2024-02-17 21:08:08.792061: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Refiner unloaded.
model_type EPS
UNet ADM Dimension 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra {'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids', 'cond_stage_model.clip_l.text_projection', 'cond_stage_model.clip_l.logit_scale'}
Base model loaded: /content/Fooocus/models/checkpoints/juggernautXL_v8Rundiffusion.safetensors
Request to load LoRAs [['sd_xl_offset_example-lora_1.0.safetensors', 0.1], ['None', 1.0], ['None', 1.0], ['None', 1.0], ['None', 1.0]] for model [/content/Fooocus/models/checkpoints/juggernautXL_v8Rundiffusion.safetensors].
Loaded LoRA [/content/Fooocus/models/loras/sd_xl_offset_example-lora_1.0.safetensors] for UNet [/content/Fooocus/models/checkpoints/juggernautXL_v8Rundiffusion.safetensors] with 788 keys at weight 0.1.
Fooocus V2 Expansion: Vocab with 642 words.
Fooocus Expansion engine loaded for cuda:0, use_fp16 = True.
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.75 seconds
App started successful. Use the app with http://127.0.0.1:7865/ or 127.0.0.1:7865 or https://dd82cd31ebbd71f74a.gradio.live
[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 4.0
[Parameters] Seed = 6557739978184862175
[Fooocus] Downloading control models ...
[Fooocus] Loading control models ...
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 30 - 15
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
[Fooocus] Processing prompts ...
[Fooocus] Preparing Fooocus text #1 ...
[Prompt Expansion] man in a street, highly detailed, cinematic, dramatic ambient light, focus, directed, perfect composition, intricate, elegant, dynamic, vibrant colors, inspired, rich deep vivid color, beautiful professional background, stunning, flawless symmetry, striking, iconic, creative, great grand, artistic, fine detail, best, awesome, colorful, atmosphere, cool, full, amazing, winning
[Fooocus] Preparing Fooocus text #2 ...
[Prompt Expansion] man in a street, cinematic, dynamic light, epic background, colorful, atmosphere, highly detailed, sharp focus, elegant, professional, fine detail, full, combined colors, perfect composition, creative, positive, unique, beautiful, attractive, confident, intricate, iconic, cute, best, new, great, novel, romantic, hopeful, artistic, color, saturated, clear
[Fooocus] Encoding positive #1 ...
[Fooocus Model Management] Moving model(s) has taken 0.12 seconds
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
[Fooocus] Image processing ...
[Parameters] Denoising Strength = 1.0
[Parameters] Initial Latent shape: Image Space (896, 1152)
Preparation time: 5.97 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 2.75 seconds
100% 30/30 [00:24<00:00,  1.23it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.33 seconds
Image generated with private log at: /content/Fooocus/outputs/2024-02-17/log.html
Generating and saving time: 30.03 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 2.00 seconds
100% 30/30 [00:25<00:00,  1.18it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.37 seconds
Image generated with private log at: /content/Fooocus/outputs/2024-02-17/log.html
Generating and saving time: 30.39 seconds
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.97 seconds
Total time: 70.76 seconds
(1006, None)

**Comment by Goliat48** [comment]
> Please restart the colab task.
> 
> Excerpt from the readme:
> 
> > Note that this Colab will disable refiner by default because Colab free's resources are relatively limited (and some "big" features like image prompt may cause free-tier Colab to disconnect). We make sure that basic text-to-image is always working on free-tier Colab.
> 
> Keep in mind that img2img and inpaint are not officially supported in Colab, but may be possible to use. There are measures to mitigate this, see #1710

Thanks for your answer Mash
I´m not using a free Colab. I´m using Colab Pro (paying it). 
I have 

> Please restart the colab task.
> 
> Excerpt from the readme:
> 
> > Note that this Colab will disable refiner by default because Colab free's resources are relatively limited (and some "big" features like image prompt may cause free-tier Colab to disconnect). We make sure that basic text-to-image is always working on free-tier Colab.
> 
> Keep in mind that img2img and inpaint are not officially supported in Colab, but may be possible to use. There are measures to mitigate this, see #1710

Thanks for your answer. Mash
I´m not using free Colab. I´m using Colab Pro (paying it ).
I´ve tried many things using high Vram but the problem is still there. Inpaint is not longer working since two days ago


(GPU) del backend de Google Compute Engine que utiliza Python 3
Mostrando recursos desde las 22:15 a las 22:51
RAM del sistema
9.8 / 51.0 GB
 
RAM de la GPU
2.6 / 16.0 GB
 
Disco
34.1 / 166.8 GB

The small symbol spins indefinitely and the Skip and Stop buttons dont work.

I´ve generated hundreds of images before using the three Inpaint alternatives without any problem, until two days ago suddenly  stop working.

**Comment by mashb1t** [comment]
The last update has been released over a week ago (11.02.), so this issue is most likely not related.
See https://github.com/lllyasviel/Fooocus/commits/main/

Error 1006 is an issue linked with swap, please investigate how this is possible to set up correctly in Colab.
https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md#error-1006
